### PR TITLE
feat(content): Add `provoke` to the Kor Efret

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
It was discussed in the past that since the Quarg are protective of the player firing on the Gegno's civilians, it would make sense the Quarg based in Efret space are also protective of a peaceful hard-life species (especially considering the Korath's history). Now that the Quarg government is split up to each relative section, this is a good time to introduce this.

This PR gives the Kor Efret a `provoke` value of 0.1, and gives the Efret Quarg a 1:1 ratio with the Efret.

This still doesn't stop the player from stealing Efret ships of course, so that is still possible for ship capturing, but now there are repercussions to doing so, which makes sense.

## Save File
N/A

### CODE REVIEW

**Comment by Saugia** [comment]
Though, Efret ships have a lot of crew, so a `provoke` value of 1 isn't much different than 10. Maybe it could go to 0.1.

File: data/governments.txt [hunk 1]
```diff
@@ -1079,12 +1079,12 @@ government "Quarg (Kor Efret)"
...
 		"Quarg (Hai)" 1
 		"Quarg (Gegno)" 1
-		"Kor Efret" .01
+		"Kor Efret" .1
 		"Kor Mereti" -.01
 		"Kor Sestor" -.01
 	"custom penalties for"
 		"Kor Efret"
-			provoke 0.01
+			provoke 0.1
```
Comment (warp-core) [inline data/governments.txt:1087]: This means the Quarg care more about the player provoking the Kor Efreti than the Kor Efreti themselves do.

Comment (Saugia) [reply data/governments.txt:1087]: How is that the case if the Efret also have a penalty of 0.1 for `provoke`?

Comment (warp-core) [reply data/governments.txt:1087]: Oh, I didn't see that part, sorry.

Why does the player need to lose reputation for provoking the Efreti?

**Comment by Saugia** [comment]
Apparently this doesn't work now, which it did for the Gegno a while ago. Gonna have to figure that out.

File: data/governments.txt (hunk 1 above)
Comment (Saugia) [reply data/governments.txt:1087]: For the same reasons as the Gegno - when I brought up that interaction with the Gegno, as in the idea that shooting them is enough to upset the very protective Quarg, a lot of creators thought the Efreti should receive the same treatment. While the Quarg don't care about humans shooting each other, or shooting the Hai, etc., they would care a lot more if the player is shooting at a peaceful, struggling species, especially one as limited as the Efreti.

**Comment by Saugia** [comment]
That should fix it. It turns out that the `provoke` works normally as per crew member with a government, but when another government is referencing the `custom penalty` of something, such as `provoke`, it's talking about the value in which it takes it at. For example:

The Efret having a `provoke` of 0.1 is 0.1. Having another gov, in this case the Efret Quarg, have a `custom penalty` `provoke` of 1, means that they are also provoked by the value of 0.1. If that penalty is 2, it would be 0.2, etc.. @warp-core

**Comment by Saugia** [comment]
Warp made a good point that the Quarg care as much about shooting the Efreti as they do destroying their ships. I've bumped down the provoke value to 0.01 instead of 0.1.

**Review by Quantumshark — APPROVED** [review]

**Pull request merged**